src/fsharp/fsc.fs

```diff
@@ -621,15 +621,13 @@ let EncodeInterfaceData(tcConfig:TcConfig,tcGlobals,exportRemapping,generatedCcu
         let resource = WriteSignatureData (tcConfig,tcGlobals,exportRemapping,generatedCcu,outfile)
         if verbose then dprintf "Generated interface data attribute!\n";
         // REVIEW: need a better test for this
-        let outFileNoExtension = Filename.chopExtension outfile
-        let isCompilerServiceDll = outFileNoExtension.Contains("FSharp.LanguageService.Compiler")
-        if (tcConfig.useOptimizationDataFile || tcGlobals.compilingFslib || isCompilerServiceDll) && not isIncrementalBuild then 
```

Comment:
See earlier comment, this is removing the special treatment of FSharp.LanguageService.Compiler